getting-started/multiband-analysis.rst

```diff
@@ -235,48 +245,66 @@ A useful flag is ``base_PsfFlux_flag``, which is the logical combination of spec
 
 .. code-block:: python
 
-   isGoodFlux = ~iSources['base_PsfFlux_flag']
+   isGoodFlux = ~iSources['base_PsfFlux_flag'] & ~rSources['base_PsfFlux_flag'] & ~gSources['base_PsfFlux_flag']
 
 Since the ``base_PsfFlux_flag`` is ``True`` for sources with measurement errors, you used the unary invert operator (``~``) so that well-measured sources are ``True`` in the ``isGoodFlux`` array.
+We need to ``and`` together the sources flagged as good between both the r-band and i-band so that we know each source has both magnitudes.
 
 Finally, combine all these boolean index arrays together:
 
 .. code-block:: python
 
-   selected = isPrimary & isStellar & isGoodFlux
+   selected_stellar = isPrimary & isStellar & isGoodFlux
+   selected_extended = isPrimary & isExtended & isGoodFlux
 
 In the next step, you'll plot a color-magnitude diagram of the sources you've selected.
 
-Plot a CMD
-==========
+Make a color-color diagram
+==========================
 
-The product of this effort will be an *r-i* CMD.
+The product of this effort will be an *g-r*/*r-i* color-color diagram showing both galaxies and stars.
 You can use matplotlib_ to create this visualization:
 
 .. code-block:: python
 
    import matplotlib.pyplot as plt
 
+   # Grab just the magnitudes and ignor the errors for now
+   plt_gMags_stellar = [el[0] for el in gMags[selected_stellar]]
+   plt_rMags_stellar = [el[0] for el in rMags[selected_stellar]]
+   plt_iMags_stellar = [el[0] for el in iMags[selected_stellar]]
+   gmr_stellar = [el1 - el2 for el1, el2 in zip(plt_gMags_stellar, plt_rMags_stellar)]
+   rmi_stellar = [el1 - el2 for el1, el2 in zip(plt_rMags_stellar, plt_iMags_stellar)]
+
+   plt_gMags_extended = [el[0] for el in gMags[selected_extended]]
+   plt_rMags_extended = [el[0] for el in rMags[selected_extended]]
+   plt_iMags_extended = [el[0] for el in iMags[selected_extended]]
+   gmr_extended = [el1 - el2 for el1, el2 in zip(plt_gMags_extended, plt_rMags_extended)]
+   rmi_extended = [el1 - el2 for el1, el2 in zip(plt_rMags_extended, plt_iMags_extended)]
+
    plt.style.use('seaborn-notebook')
    plt.figure(1, figsize=(4, 4), dpi=140)
-   plt.scatter(rMags[selected][0] - iMags[selected][0],
-               iMags[selected][0],
-               edgecolors='None', s=2, c='k')
-   plt.xlim(-0.5, 3)
-   plt.ylim(25, 14)
-   plt.xlabel('$r-i$')
-   plt.ylabel('$i$')
+   plt.scatter(gmr_stellar,
+               rmi_stellar,
+               edgecolors='None', s=4, c='k')
+   plt.scatter(gmr_extended,
+               rmi_extended, marker='v',
+               edgecolors='None', s=4, c='r')
+   plt.xlabel('$g-r$')
+   plt.ylabel('$r-i$')
    plt.subplots_adjust(left=0.125, bottom=0.1)
    plt.show()
 
 You should see a figure like this:
 
-.. figure:: multiband-analysis-stars.png
-   :alt: r-i color-magnitude diagram of stars.
+.. figure:: multiband-analysis.png
+   :alt: color-color plot for stars and galaxies.
```

Comment:
It is odd to me seeing stars in black rather than blue but don't think that is particularly important.